src/app/inventory/store/stats.ts

```diff
@@ -292,14 +306,31 @@ function enhanceStatsWithPlugs(
   return stats;
 }
 
+function buildBaseStats(stats: D2Stat[], sockets: DimSocket[]) {
+  for (const socket of sockets) {
+    if (socket.plug && socket.plug.plugItem.investmentStats) {
+      for (const perkStat of socket.plug.plugItem.investmentStats) {
```

Comment:
Probably worth at least mentioning in a comment that this relies on armor stats never being interpolated.

Reply:
fair